build/commands/lib/config.js

```diff
@@ -788,28 +755,20 @@ Config.prototype.buildArgs = function () {
   return args
 }
 
-Config.prototype.shouldSign = function () {
-  if (this.skip_signing || this.isComponentBuild() || this.targetOS === 'ios') {
-    return false
-  }
-
-  if (this.targetOS === 'android') {
-    return this.braveAndroidKeystorePath !== undefined
-  }
-
-  if (this.targetOS === 'mac') {
-    return this.mac_signing_identifier !== undefined
+Config.prototype.maybeSkipSigning = function () {
+  if (this.skip_signing != null) {
+    return this.skip_signing
   }
 
   if (process.platform === 'win32') {
     return (
-      process.env.CERT !== undefined
-      || process.env.AUTHENTICODE_HASH !== undefined
-      || process.env.SIGNTOOL_ARGS !== undefined
+      process.env.CERT == null
```

Comment:
these should be converted to gn args so we can move all of this logic to config.gni